packages/create-app/src/index.ts

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import program from 'commander';
+import chalk from 'chalk';
+import { exitWithError } from './lib/errors';
+import { version } from './lib/version';
+
+const main = (argv: string[]) => {
+  program.name('backstage-create-app').version(version);
+
+  program
+    .description('Creates a new app in a new directory')
+    .option(
+      '--skip-install',
+      'Skip the install and builds steps after creating the app',
+    )
+    .action(
+      lazyAction(() => import('./createApp'), 'default'),
```

Comment:
No need for the lazy load here, It's done in the CLI because loading in things like `webpack` takes a long time, and it's not used by all commands. Here we have a single command so it's fine to just import and call directly.